packages/datadog-plugin-google-cloud-pubsub/test/index.spec.js

```diff
@@ -0,0 +1,266 @@
+'use strict'
+
+const agent = require('../../dd-trace/test/plugins/agent')
+const { expectSomeSpan, withDefaults } = require('../../dd-trace/test/plugins/helpers')
+const plugin = require('../src')
+const id = require('../../dd-trace/src/id')
+
+const BASE_PROJECT_ID = `test-project-${id()}-`
+let projectCounter = 0
```

Comment:
I'd still prefer to remove these to keep it simple. Performance of calculating a new ID on every test is minimal.